### ISSUE TITLE
module 'keras.utils' has no attribute 'get_file'

### ISSUE THREAD

**Issue by deevyankar** [ZFTurbo/efficientnet_3D#6]
model = efn.EfficientNetB0(input_shape=(64, 64, 64, 3), weights='imagenet')   in this line

**Comment by AmokraneMancer** [comment]
I had the same issue. solved by using `import efficientnet_3D.tfkeras as efn` instead of `import efficientnet_3D.keras as efn`

**Comment by deevyankar** [comment]
Thanks  its working now.

**Comment by arubiales** [comment]
An easy patch I did  in  `__init__.py` file, change the function `inject_keras_modules` importing directly from `Tensorlfow` instead of `Keras` because `get_file` is a function that only exist in `tensorflow.keras` not in `keras`.

You must change:  `import keras` for `import tensorflow.keras as keras`. The new function will be:

```
def inject_keras_modules(func):
    import tensorflow.keras as keras
    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        kwargs['backend'] = keras.backend
        kwargs['layers'] = keras.layers
        kwargs['models'] = keras.models
        kwargs['utils'] = keras.utils
        return func(*args, **kwargs)

    return wrapper
```

As I said at the beggining of this comment, this is just a patch

**Comment by deevyankar** [comment]
Friends I have another issue,
base_model = efn.EfficientNetB0(input_shape=(182,218,182,1), weights='imagenet',classes=2) showing error  [ **its working only for 3 number of channels**]

Downloading data from https://github.com/ZFTurbo/efficientnet_3D/releases/download/v1.0.0/efficientnet-b0_inp_channel_3_tch_0_top_False.h5
19546112/19540520 [==============================] - 0s 0us/step
19554304/19540520 [==============================] - 0s 0us/step
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-721119ae2919> in <module>()
----> 1 base_model = efn.EfficientNetB0(input_shape=(182,218,182,1), weights='imagenet',classes=2)

7 frames
<__array_function__ internals> in transpose(*args, **kwargs)

/usr/local/lib/python3.7/dist-packages/numpy/core/fromnumeric.py in _wrapfunc(obj, method, *args, **kwds)
     56 
     57     try:
---> 58         return bound(*args, **kwds)
     59     except TypeError:
     60         # A TypeError occurs if the object does have such a method in its

ValueError: axes don't match array

**Comment by atamazian** [comment]
That's right, because pretrained model  has 3 channels, and your model has 1 channel. You can't use these weights with your model.

**Comment by atamazian** [comment]
I suggest to close this issue since it have been solved.
